zone/gm_commands/summonburiedplayercorpse.cpp

```diff
@@ -3,26 +3,34 @@
 
 void command_summonburiedplayercorpse(Client *c, const Seperator *sep)
 {
-	Client *t = c;
-
+	auto target = c;
 	if (c->GetTarget() && c->GetTarget()->IsClient() && c->GetGM()) {
-		t = c->GetTarget()->CastToClient();
-	}
-	else {
-		c->Message(Chat::White, "You must first select a target!");
-		return;
+		target = c->GetTarget()->CastToClient();
 	}
 
-	Corpse *PlayerCorpse = database.SummonBuriedCharacterCorpses(
-		t->CharacterID(),
-		t->GetZoneID(),
+	auto corpse = database.SummonBuriedCharacterCorpses(
```

Comment:
Small thing, but doesn't hurt to keep the pointer `auto *corpse` because it prevents a copy. Doesn't make a huge material impact here but good practice to not copy when you don't have to especially in hotter paths

Reply:
👍